src/loaders/med4_load.c

```diff
@@ -546,9 +546,13 @@ static int med4_load(struct module_data *m, HIO_HANDLE *f, const int start)
 			} else if (_type == -1) {
 				hio_seek(f, 20, SEEK_CUR);
 				num_smp += hio_read16b(f);
+				_len -= 22;
 			}
 
-			hio_seek(f, _pos + _len, SEEK_SET);
```

Comment:
What are we doing with the `_pos` variable we read above now?

Reply:
Good catch, that variable isn't needed anymore. I'll push a patch to remove it soon.